tests/block/009

```diff
@@ -25,7 +25,7 @@ DESCRIPTION="check page-cache coherency after BLKDISCARD"
 . common/scsi_debug
 
 requires() {
-	_have_scsi_debug && _have_program xfs_io
+	_have_scsi_debug && _have_program xfs_io && _have_program blkdiscard
```

Comment:
blkdiscard is part of util-linux, which we already expect to be installed. Or are there versions of util-linux without blkdiscard floating around?

Reply:
thanks for the feedback, Omar. I run blktests from an initramfs image ( https://github.com/rapido-linux/rapido/blob/master/cut_blktests_zram.sh and cut_blktests_rbd.sh ), so this was just intended as a reminder. I'll drop it along with the blockdev one below.